controllers/petitions.controller.js

```diff
@@ -68,61 +68,75 @@ petitionsController.createWithFund = catchAsync(async (req, res, next) => {
 
 petitionsController.createPetitionWithItems = catchAsync(
   async (req, res, next) => {
-    let { phone, firstName, itemArray, petitionType, description } = req.body;
+    let {
+      phone,
+      firstName,
+      lat,
+      lng,
+      itemArray,
+      petitionType,
+      description,
+    } = req.body;
     if (!petitionType) {
       return next(new AppError(400, "Required fields are missing!"));
     }
+    console.log(lat, lng);
```

Comment:
Could we remove this to keep out logs quieter?

Reply:
ok, thanks, I'll do it